### ISSUE TITLE
best way to save face encoding  

### ISSUE THREAD

**Issue by syedmustafa54** [deepinsight/insightface#850]
I am trying to save 1m face encoding

**Comment by Akshaysharma29** [comment]
@syedmustafa54  have figured it out?

**Comment by syedmustafa54** [comment]
@Akshaysharma29 i am using mongobd it’s taking 15ms to querying 2m items.

**Comment by Akshaysharma29** [comment]
@syedmustafa54 by 2m you mean 2 million?

**Comment by syedmustafa54** [comment]
Yes

**Comment by Akshaysharma29** [comment]
Okay Thanks @syedmustafa54. Have you tried hnswlib?

**Comment by syedmustafa54** [comment]
Yes I have heard of it but didn’t use it I have tried faiss it’s good it will use gpu you can give it a try and compare both. You can join me on discord or telegram for instant reply 🙂

**Comment by Akshaysharma29** [comment]
Thanks @syedmustafa54 but I am not on any of this app can we connect on linkedIn?

**Comment by Etty-Cohen** [comment]
@tomriddle54 , @Akshaysharma29 , Did you solve the problem? I would like for you to share how

**Comment by Etty-Cohen** [comment]
@tomriddle54 , @Akshaysharma29 , Did you solve the problem? I would like for you to share how
